src/libraries/System.Security.Cryptography.Xml/tests/SignatureSupport.cs

```diff
@@ -0,0 +1,20 @@
+namespace System.Security.Cryptography.Xml.Tests
+{
+    class SignatureSupport
+    {
+        private static int _supportsRsaSha1Signatures = 0;
+
+        public static bool SupportsRsaSha1Signatures
+        {
+            get
+            {
+                if (_supportsRsaSha1Signatures == 0)
+                {
+                    bool supported = System.Security.Cryptography.Tests.SignatureSupport.CanProduceSha1Signature(new RSACryptoServiceProvider());
+                    _supportsRsaSha1Signatures = supported ? 1 : -1;
+                }
+                return _supportsRsaSha1Signatures == 1;
+            }
+        }   
+    }
+}
```

Comment:
Nit: Blank lines at the end of the file are recommended.

https://github.com/dotnet/runtime/blob/fff4ff0913a2396cfae61c701f2b59ffe5e27b49/.editorconfig#L9-L10

```suggestion
}

```